invenio_pidstore/provider.py

```diff
@@ -48,6 +48,14 @@ class PIDStatus:
     registered.
     """
 
+    @classmethod
+    def list_in_order(cls):
```

Comment:
I can, but then you'll have to call it from an instance of PIDStatus here: https://github.com/krzysztof/invenio-pidstore/commit/d606aea905f9786f8a14d6058535704b30b6e959#diff-4788ef5235e6aaf66909f08dd7e63bb5R36 would it be better ?

Btw. I didn't use Enum because we refer to the fields in this class almost exclusively when comparing it with the value in the database. If I had used Enum I'd have to refer to the value property ``orm_instance.status = PIDStore.NEW.value`` instead of simpler ``orm_instance.status = PIDStore.NEW```.